config/schema/og.schema.yml

```diff
@@ -24,7 +24,7 @@ og.settings:
     groups:
       type: mapping
       mapping:
-        bundles:
+        node:
```

Comment:
I think we need to tweak the schema for the entity type id keys, but not hard code them like this.